src/ast/mod.rs

```diff
@@ -6373,6 +6472,76 @@ impl Display for MySQLColumnPosition {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
```

Comment:
Can we please add comments for these newly added pub structs (ideally with a link back to the manual of whatever dialect they come from?)

Reply:
done